src/domain/dao.ts

```diff
@@ -6,6 +6,13 @@ export function getDAO(id: string): DAO {
   let dao = store.get('DAO', id) as DAO;
   if (dao == null) {
     dao = new DAO(id);
+    dao.name = '';
```

Comment:
why is this needed?

Reply:
because the vault event happens before the dao creation event, and if it will be null the sbgraph will crash as it is a non null field